README.md

```diff
@@ -404,6 +412,7 @@ A list of **Free** and **Open Source Software** ***(FOSS)*** for **Android** –
 * [**SongTube**](https://github.com/SongTube/SongTube-App)
 * [**raag**](https://github.com/raag-music/raag) <sup>**[[F-Droid](https://f-droid.org/app/in.amfoss.raag)]**</sup>
 * [**Music**](https://github.com/z-huang/music)
+* [**ViMusic**](https://github.com/vfsfitvnm/ViMusic)
```

Comment:
Addendum: Let's go for F-Droid :crossed_fingers: https://github.com/vfsfitvnm/ViMusic/issues/32